docs/installation.md

```diff
@@ -102,4 +102,29 @@ Note, the permissions on this folder need to allow the container to read, write,
 
 ## Manual Install
 
-Coming soon
+=== "Windows"
+    On Windows, there are a couple of extra steps involved.  
+
+    1. [Download and install Python](https://www.python.org/downloads/). This is the framework Kapowarr runs on top of.  
+       _Make sure you select to add Python to PATH when prompted. This will make installing requirements much easier._
+    2. Download (or clone) the [latest Kapowarr release](https://github.com/Casvt/Kapowarr/releases/latest).  
+    3. Extract the zip file to a folder on your machine.  
+       We suggest something straightforward - `C:\services\Kapowarr` is what we'll use as an example.
+    4. Install the required python modules (found in `requirements.txt`).
+       This can be achieved from a command prompt, by changing to the folder you've extracted Kapowarr to and running a python command.
+       ```
```

Comment:
Change `\`\`\`` to something like `\`\`\`ps` to get proper color coding of the commands inside according to the language. Check if it's 'ps' for Powershell, I don't know for sure.

I won't leave a comment for all places that this occurs, so check yourself.

Reply:
Adjusted.